modules/ui/maproulette_editor.js

```diff
@@ -185,49 +198,32 @@ export function uiMapRouletteEditor(context) {
       .append('div')
       .attr('class', 'detail-section')
       .merge(detailSection);
-
```

Comment:
This is kind of a nit, but the code is easier to read with the empty line after the d3 statement, because the d3 idiom is to method chain a bunch of things together, and the blank line makes it clear when the chain ends.